backend/src/test/java/com/github/martinhermes86/backend/controller/PlantControllerTest.java

```diff
@@ -141,4 +141,77 @@ void deletePlantById_when_found() throws Exception {
         mvc.perform(MockMvcRequestBuilders.delete("/api/plants/" + actual.id()))
                 .andExpect(status().isOk());
     }
+
+    @Test
+    void updatePlant_when_not_found() throws Exception {
```

Comment:
just a small suggestion, it would be more readable if the name of the test includes the result you expect. for example:
testUpdatePlant_returnsBlaBla_whenPlantNotFound